src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -436,10 +436,7 @@ private Autodesk.Revit.DB.Parameter GetParameterByName(string parameterName)
             // 3. If it exist: Use the first writable parameter
             // 4. Otherwise: Use the first read-only parameter
             //
-            var allParams =
-            InternalElement.Parameters.Cast<Autodesk.Revit.DB.Parameter>()
-                .Where(x => string.CompareOrdinal(x.Definition.Name, parameterName) == 0)
-                .OrderBy(x => x.Id.Value);
+            var allParams = InternalElement.GetParameters(parameterName).OrderBy(x => x.Id.Value);
```

Comment:
cool, so if I understand correctly, you skip the dynamo function and use the RevitAPI - this is usually a good pattern for sure, is there anything weird with element binding this could cause?

Reply:
No, I don't believe so.